CUDAEnsemble error handling details.

Robadob opened the pull request:
Relates https://github.com/FLAMEGPU/FLAMEGPU2/pull/839

ptheywood reviewed (changes requested):
It would be useful to state what the default error level is (Slow) if not provided via the cli or set programatically in the config by the user.

Robadob commented:
> It would be useful to state what the default error level is (Slow) if not provided via the cli or set programatically in the config by the user.

Will do. I noticed that was missing from all the docstring of command line args when addressing your code comments.
